trino/auth.py

```diff
@@ -131,3 +132,38 @@ def get_exceptions(self):
 
     def handle_error(self, handle_error):
         pass
+
+
+class BearerAuth(AuthBase):
+    """
+    Custom implementation of Authentication class for bearer token
+    """
+    def __init__(self, token):
+        self.token = token
+
+    def __call__(self, r):
+        r.headers["authorization"] = "Bearer " + self.token
```

Comment:
I don't think it matters, but the header is normally capitalized https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization